app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java

```diff
@@ -22,16 +22,20 @@
 
     public static final String USERNAME = "Username";
     public static final String PASSWORD = "Password";
+    public static String NOTIFICATION = "";
```

Comment:
@georgezsiga add some string content here (this will be used as a key in the sharedpref), unless this wont work